### DISCUSSION TITLE
Benefits not yet activated for a week.

### DISCUSSION THREAD

**Discussion by zacwm** [community/community#128742] [GitHub Education]
### Hi everyone,

Hey everyone!

I've accidentally let my student benefits expire, it's been pretty hectic lately.

I've reapplied and got approved on the 9th of June, and as of tonight (16th of June as of writing this) it is still saying my benefits are still not active.
Upon reading the FAQ, It suggested I should start this discussion if the benefits have not been activated within 5 days. It's now over that so here I am. 😛

I did also see that during busy times it may take longer, so sorry if that's the case haha.
Thanks!

**Comment by davevad93** [comment]
Hi @zacwm ,

Don't worry in a few days you'll surely have your benefits. Take a look this discussion #128457 , this user had to wait almost 10 days but in the end everything turned out fine.

**Comment by samus-aran** [comment]
Hi @zacwm 🎉 

It appears that your benefits have been granted. Thank you for your patience. Let us know if you have any difficulties redeeming them.
